mlflow/onnx.py

```diff
@@ -0,0 +1,175 @@
+from __future__ import absolute_import
+
+import sys
+
+if sys.version_info <= (3, 6):
+    print('Requires Python 3.6')
+    sys.exit(1)
+
+import os
```

Comment:
It's kind of gross, but it seems the best solution to eliminating linter errors here is to include a "  # noqa" or "  #  pylint:disable=E402" after each import line as described in https://github.com/PyCQA/pycodestyle/issues/264

Reply:
Note that you'll also want to ensure these tests run in Travis only against Python 3. You can do this by 1) marking the tests as large via the `@pytest.mark.large` annotation before the test declaration (see [here](https://github.com/mlflow/mlflow/blob/7d71ff3894699b5800e52080c8c0f6fe5d46832e/tests/pyfunc/test_spark.py#L68) for an example) and 2) adding a line to the end of .travis.yml (under the "script" section) that looks like:

```
  - if [[ "$TRAVIS_PYTHON_VERSION" == "3.6" ]]; then
      pytest --verbose tests/onnx --large
    fi 
```